src/app/modules/item/services/item-task-answer.service.ts

```diff
@@ -32,13 +33,16 @@ export class ItemTaskAnswerService implements OnDestroy {
   private taskToken$ = this.taskInitService.taskToken$.pipe(takeUntil(this.error$));
 
   private initialAnswer$: Observable<Answer | null> = this.config$.pipe(
-    switchMap(({ route, attemptId, shouldReloadAnswer }) => (
-      shouldReloadAnswer ? this.currentAnswerService.get(route.id, attemptId) : of(null)
-    )),
-    catchError(error => {
-      // currently, the backend returns a 403 status when no current answer exist for user+item+attempt
-      if (errorIsHTTPForbidden(error)) return of(null);
-      throw error;
+    switchMap(({ route, attemptId, shouldReloadAnswer }) => {
+      if (!shouldReloadAnswer) return of(null);
+      if (route.answerId) return this.getAnswerService.get(route.answerId);
```

Comment:
I suppose the error handling is missing here. If you give your link to another user and so do not have access to the answer (or just no access for another reason), we should decide what happens...
Intuitively, I would say (may require more thoughts) that it should display an error message such as `Not able to load the answer <a same page without answerId>load your most recent answer</a>.`

Reply:
fixed